packages/create-analog/index.js

```diff
@@ -354,6 +355,10 @@ function addDevDependencies(pkg) {
   );
 }
 
+function addYarnDevDependencies(pkg) {
+  pkg.devDependencies['@angular-devkit/build-angular'] = ['^17.3.5'];
```

Comment:
This should be based on the template used also. If its angular-v16, it should add the v16 version